config/admin/admin.go

```diff
@@ -494,8 +499,50 @@ func init() {
 	)
 	user.EditAttrs(user.ShowAttrs())
 
+	argumentResource := Admin.NewResource(&FromToArgument{})
+	argumentResource.Meta(&admin.Meta{
+		Name: "From",
+		Type: "select_one",
+		Valuer: func(_ interface{}, context *qor.Context) interface{} {
+			locale, _ := context.GetDB().Get("l10n:locale")
+			return locale.(string)
+		},
+		Collection: func(value interface{}, context *qor.Context) (results [][]string) {
+			names := []string{"United States", "United Kingdom"}
+			for _, name := range names {
+				results = append(results, []string{name, name})
+			}
+			return
+		},
+	})
+	argumentResource.Meta(&admin.Meta{
+		Name: "To",
+		Type: "select_many",
+		Valuer: func(_ interface{}, context *qor.Context) interface{} {
+			return []string{""}
+		},
+		Collection: func(value interface{}, context *qor.Context) (results [][]string) {
+			// ..........
+			return
+		},
+	})
+
+	// Blog Management
+	article := Admin.AddResource(&models.Article{}, &admin.Config{Menu: []string{"Blog Management"}})
+	article.IndexAttrs("ID", "VersionName", "ScheduledStartAt", "ScheduledEndAt", "Author", "Title")
+	// article.Meta(&admin.Meta{
```

Comment:
If we uncomment this section and change the meta from `Name: "Store",` to `Name:  "StoreID", Label: "Store",`, it will also work.